Orange/preprocess/reprable.py

```diff
@@ -0,0 +1,39 @@
+class Reprable:
+    def __repr__(self):
+        """ A helper class that, when inherited, creates an executable __repr__
+        String based on the arguments passed to the preprocessor's __init__ function.
```

Comment:
I wouldn't say this is necessarily for preprocessors only. Approximately the same code can be used for nearly arbitrary objects we have, including `Learner` and `Model` objects?